src/java/org/apache/cassandra/db/compaction/CompactionRealm.java

```diff
@@ -79,6 +79,11 @@ default String getKeyspaceName()
         return metadata().keyspace;
     }
 
+    /**
+     * @return the replication factor for this table.
+     */
+    int getKeyspaceReplicationFactor();
```

Comment:
Do all tables in a keyspace have the same replication factor? I expected this method to be called `getTableReplicationFactor`.

Reply:
replication factor is keyspace property, so all tables in a keyspace have the same replication factor

Reply:
RF is a complicated thing that can't just be a single int.  Should this be "getKeyspaceReplicationFactorInCurrentDC" or something?  At the least we should update the comment to say what is meant.  Is this the sum of all replication factors across all DC's or only the replication factor in the current DC.